src/main/java/edu/wpi/first/shuffleboard/components/TilePane.java

```diff
@@ -29,7 +31,7 @@
 
   private static final int DEFAULT_COL_COUNT = 1;
   private static final int DEFAULT_ROW_COUNT = 1;
-  private static final double MIN_TILE_SIZE = 32; // pixels
+  private static final double MIN_TILE_SIZE = 64; // pixels
```

Comment:
Number of pixels is a bad size metric as it can vary wildly. Shouldn't you use `em` instead?
https://stackoverflow.com/a/609561/3708426